Module.yaml

```diff
@@ -0,0 +1,91 @@
+version: 1.0.0
+name: Melita IO Batch for devices controller
```

Comment:
Name is too long to display in UI drag-drop block. Maybe just keep `Melita IO Batch`

Reply:
set it to Melita IO Batch